hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/src/test/java/org/apache/hadoop/mapreduce/lib/input/TestCombineFileInputFormat.java

```diff
@@ -1831,7 +1831,7 @@ private void splitRealFiles(String[] args) throws IOException {
     }
   }
 
-  public static void main(String[] args) throws Exception{
+  /*public static void main(String[] args) throws Exception{
```

Comment:
Leave the main entry point here?

Reply:
Thank you for helping review the code! I will continue to improve this part of the code.